CONTRIBUTING.md

```diff
@@ -186,7 +184,7 @@ We recommend a read on our [Collaborator Guide](COLLABORATOR_GUIDE.md#accepting-
     - The person that is fast-tracking the PR (adding the label) must also comment on the PR that they're requesting the PR to be fast-tracked
     - The comment must mention `@nodejs/website` and must have at least one 👍 (or any other sort of approval reaction) if the person fast-tracking the PR is the author of the PR.
     - Fast-tracking is only allowed for small bug fixes, small feature changes, localisation changes, or other sorts of non-critical/highly-impacting changes that are not covered by the previous rule that allows PRs to be merged immediately.
-    - Fast-tracking cannot be used for updates on the `COLLABORATOR_GUIDE.md`,  CONTRIBUTING.md` guide, `CODEOWNERS`, GitHub Actions or any security-impacting file or document that changes the governing policies of this repository.
+    - Fast-tracking cannot be used for updates on the `COLLABORATOR_GUIDE.md`, CONTRIBUTING.md`guide,`CODEOWNERS`, GitHub Actions or any security-impacting file or document that changes the governing policies of this repository.
```

Comment:
the backticks on this file are wrong, leading to incorrect rendering of the inline code block

![image](https://github.com/nodejs/nodejs.org/assets/298435/99171fa4-dcb5-498b-80f9-7e22d2f37b2e)

```suggestion
    - Fast-tracking cannot be used for updates on the `COLLABORATOR_GUIDE.md`, `CONTRIBUTING.md` guide, `CODEOWNERS`, GitHub Actions, or any security-impacting file or document that changes the governing policies of this repository.
```